cypress/integration/fee-fine/newFeeFines.spec.js

```diff
@@ -0,0 +1,117 @@
+import uuid from 'uuid';
+import TestType from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import getRandomPostfix from '../../support/utils/stringTools';
+import Users from '../../support/fragments/users/users';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import AppPaths from '../../support/fragments/app-paths';
+import UsersCard from '../../support/fragments/users/usersCard';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import UserAllFeesFines from '../../support/fragments/users/userAllFeesFines';
+import PayFeeFaine from '../../support/fragments/users/payFeeFaine';
+
+describe('Fee/fine management', () => {
+  const testData = {};
+  beforeEach(() => {
+    cy.getAdminToken();
+    const patronGroupName = `autotestPatronGroup${getRandomPostfix()}`;
+    PatronGroups.createViaApi(patronGroupName).then(patronGroupId => {
+      testData.patronGroupId = patronGroupId;
+      testData.userProperties = { barcode : uuid() };
+      Users.createViaApi({ ...Users.defaultUser, patronGroup: patronGroupId, barcode: testData.userProperties.barcode }).then(userProperties => {
+        testData.userProperties.id = userProperties.id;
+        testData.userProperties.lastName = userProperties.lastName;
+        testData.userProperties.firstName = userProperties.firstName;
+        testData.userProperties.middleName = userProperties.middleName;
+        ServicePoints.getViaApi({ limit: 1, query: 'pickupLocation=="true"' })
+          .then((res) => {
+            testData.servicePointId = res[0].id;
+            UserEdit.addServicePointViaApi(testData.servicePointId, testData.userProperties.id);
+            UsersOwners.createViaApi({ owner: uuid() }).then(owner => {
+              testData.owner = { id : owner.id, name : owner.ownerName };
+              ManualCharges.createViaApi({ ...ManualCharges.defaultFeeFineType, ownerId: owner.id }).then(manualCharge => {
+                testData.feeFineType = { id:  manualCharge.id, feeFineTypeName: manualCharge.feeFineType };
+                PaymentMethods.createViaApi(testData.owner.id).then(createdPaymentMethod => {
+                  testData.paymentMethod = { id: createdPaymentMethod.id, name: createdPaymentMethod.name };
+                  // TODO: clarify why initial login is needed to load accordion with Fee/Fines
+                  // cy.loginAsAdmin({ path: AppPaths.getUserPreviewPath(testData.userId), waiter: UsersCard.waitLoading });
```

Comment:
Do we need this comment?

Reply:
yes, there are few questions/problems which should be clarified in the future